packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -1,55 +1,46 @@
-var fs = Npm.require('fs');
-var path = Npm.require('path');
+import fs from 'fs';
+import path from 'path';
```

Comment:
I would be more specific:
```js
import { readFile } from "fs";
import { ??? } from "path";
```
It looks like you're not using `path` anywhere in this file, so I think you can remove that line.